docs/contributor/04-20-release.md

```diff
@@ -49,6 +49,6 @@ To create a release, follow these steps:
 
 To regenerate an existing release, perform the following steps:
 
-1. Delete the GitHub release
-2. Delete the GitHub tag
-3. Run the [**Create release**](#create-a-release) pipeline
+1. Delete the GitHub release.
+2. Delete the GitHub tag.
+3. Run the [**Create release**](#create-a-release) pipeline.
```

Comment:
```suggestion
3. Run the [**Create a release**](#create-a-release) pipeline.
```

Reply:
Actually, this is a UI element. I'd rather keep it as it is.
![Screenshot 2024-04-03 at 10 29 21](https://github.com/kyma-project/kyma-environment-broker/assets/107463569/f7bc6002-cffc-45fc-abdb-e19ca6490533)